### DISCUSSION TITLE
action taking long time after reticulating spines. no reason showing.

### DISCUSSION THREAD

**Discussion by StigmaCat** [Atlas-OS/Atlas#1303] [Help & Support] [answered]
action taking long time after reticulating spines. no reason showing.

[GUI | 15:12:07] Reticulating splines...
[GUI | 15:22:07] Action taking a long time...
![image](https://github.com/user-attachments/assets/9a3bd887-5561-49d5-829a-95a7e0debad6)
![image](https://github.com/user-attachments/assets/045cb879-d38a-4266-8c0f-8ac3006b183d)
new windows install. followed the whole doc. what do i do

**Comment by nafi167** [comment]
I have the samee problem. So, did you fix it later?

**Reply by StigmaCat** [reply]
kinda, by reinstalling windows 10 :) hope that helps you

**Comment by WarmXR** [comment]
same issue as well

**Comment by StigmaCat** [comment, marked as answer]
WORKED AFTER REINSTALLING WINDOWS 10. cat out my friends
